xml/src/main/java/org/ehcache/config/xml/XmlConfiguration.java

```diff
@@ -156,7 +157,21 @@ private void parseConfiguration()
           cacheClassLoader = ClassLoading.getDefaultClassLoader();
         }
       }
-      
+      if(cacheDefinition.persistenceMode() != null) {
+        CacheConfiguration.PersistenceMode mode = null;
+        switch (cacheDefinition.persistenceMode()) {
+        case CREATE_IF_ABSENT: 
+          mode = CacheConfiguration.PersistenceMode.CREATE_IF_ABSENT ;
+          break;
+        case EXPECT_EXISTS:
+          mode = CacheConfiguration.PersistenceMode.EXPECT_EXISTS;
+          break;
+        default:
```

Comment:
I'd prefer to see "case SWAP:" and have the default throw IllegalArgumentException.